.github/workflows/test.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Export environment variables
         run: |
           echo "SECRET_KEY=$(openssl rand -hex 32)" > .env
+          echo "SMTP_HOST=smtp.gmail.com" >> .env
+          echo "SMTP_PORT=465" >> .env
+          echo "EMAIL_SENDER=test@kernelci.org" >> .env
+          echo "EMAIL_PASSWORD=random" >> .env
```

Comment:
On a side topic, one thing I've found is that the .env file needs to be populated by hand when running the tests locally.  As these aren't special values but can be anything arbitrary, I think it would be better to have them supplied directly when running the test instead.

Reply:
Yes, I want to override the `Settings` class for unit tests. Something like described here: https://fastapi.tiangolo.com/advanced/settings/#settings-and-testing

Reply:
I have added env variables to `pytest` command. Also, created a follow-up PR for `Settings` class.